src/ape/managers/compilers.py

```diff
@@ -95,6 +105,14 @@ def compile(self, contract_filepaths: List[Path]) -> Dict[str, ContractType]:
 
                 contract_types_dict[contract_type.name] = contract_type
 
+        # Fix source IDs for when compiler did not account for base_path,
+        # such as solidity<0.6.9.
```

Comment:
Is there a way for this to become contained in the solidity plugin?

Reply:
yes I found a way